test/utils/test_sampling.py

```diff
@@ -308,7 +308,7 @@ def test_find_interior_point(self):
         A = np.array([[-1.0]])
         b = np.array([-3.0])
         x = find_interior_point(A=A, b=b)
-        self.assertAlmostEqual(x.item(), 6.201544, places=4)
+        self.assertAlmostEqual(x.item(), 5.0, places=4)
```

Comment:
If any `x>=3` is okay, should we just check `x>=3` rather than a specific value?

Reply:
Yeah that would work as well and would not have resulted in a test failure. It's kind of nice to know about different behaviors that may occur under different versions, ideally the behavior of our functions is the same independent of the versions of the libraries used (so long as they are compatible). So far this hasn't been much of an issue so I'd like to to the value comparison; if this gets too noisy we can chance that policy.